lib/utils/locatestrategy.ts

```diff
@@ -19,31 +19,31 @@ class LocateStrategy {
     };
   }
```

Comment:
We should use `enum` instead since it provides a concise and meaningful way to represent a related set of constants.

```typescript
enum LocateStrategyType {
  ID = 'id',
  CSS_SELECTOR = 'css selector',
  LINK_TEXT = 'link text',
  PARTIAL_LINK_TEXT = 'partial link text',
  TAG_NAME = 'tag name',
  XPATH = 'xpath',
  NAME = 'name',
  CLASS_NAME = 'class name',
  ACCESSIBILITY_ID = 'accessibility id',
  ANDROID_UIAUTOMATOR = '-android uiautomator',
  IOS_PREDICATE_STRING = '-ios predicate string',
  IOS_CLASS_CHAIN = '-ios class chain'
}
```
then we can use later as below
```typescript
  static isValid(strategy: string): boolean {
    return Object.values(LocateStrategyType).includes(strategy.toLowerCase());
  }
 ```

Reply:
I have made this change, however, on hovering over `strategy.toLowerCase()` there exists this error:
Argument of type 'string' is not assignable to parameter of type 'Strategies'.ts(2345)